### PR TITLE
Create quickmultiply.py

### PR DESCRIPTION
**Pull request by Rohit-beep-droid**
### **Describe your change:**

Made a simple multiplication function which returns the multiplication of two integers

* [ ] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [ ] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: maths/quickmultiply.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+>>> quick_multiply(0,0)
+0
+"""
+
+
+def quick_multiply(num: int, num2: int) -> int:
+  return num*num2
+
+
+def test_quick_multiply()
```
Comment (cclauss) [inline maths/quickmultiply.py:19]:
```suggestion
def test_quick_multiply():
```

File: maths/quickmultiply.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+
+def test_quick_multiply():
+"""
+>>> test_quick_multiply()
+"""
+assert -25 == test_quick_mutiply(-5,5)
+assert 1000 == test_quick_mutiply(-10,-10)
+assert 0 == test_quick_multiply(0,1000)
+assert 81 == test_quick_multiply(9,9)
```
Comment (mrmaxguns) [inline maths/quickmultiply.py:26]: The error is due to invalid indentation. Note that I added two spaces of indentation to be consistent with the rest of the file, but 4 spaces of indentation throughout the whole file are preferred.

```suggestion
  """
  >>> test_quick_multiply()
  """
  assert -25 == test_quick_mutiply(-5,5)
  assert 1000 == test_quick_mutiply(-10,-10)
  assert 0 == test_quick_multiply(0,1000)
  assert 81 == test_quick_multiply(9,9)
```

**Review by mrmaxguns — CHANGES_REQUESTED** [review]
Hello! The error is due to invalid indentation. A fix is suggested in the review. After fixing the error, you may want to run Black in order to format the code according to best practices.

I actually don't believe that this PR should be merged, though. The function is simply a wrapper of a Python operator, although one could make an argument that it is still an algorithm. I'd like to hear your thoughts on this.

**Comment by cclauss** [comment]
Closed as this is not an algorithm but a how-to-use as discussed in CONTRIBUTING.md.

**Pull request closed without merging**